types/compat/ip_address.pp

```diff
@@ -1 +1,2 @@
+# Validate an IP address
```

Comment:
I thougt this always needs to be prefixed with `# @summary`. does it then render differently?

Reply:
Does not change the rendering, but adding them makes me spot some inconsistencies…

Reply:
Well the summary line appear in the TOC but does not appear in the actual section if there is some content. It is acceptable I guess and repeating the info twice is this case seems redundant.